### ISSUE TITLE
can one specify parts of the model that are non differentiable?

### ISSUE THREAD

**Issue by jakubMitura14** [google/evojax#41]
I have a model in Jax (convolutional neural network with some modifications) where most is  fully differentiable, but parts are not - can I mark somehow which parts are not differentiable so to have correct gradient backpropagation, or it is done automatically?

Thanks!

**Comment by jakubMitura14** [comment]
Thank you! I will experiment on this, and not to clutter your issues close for now, in case of further problems will reopen.
And once more time thanks for publishing this fantastic work !!
